### PR TITLE
Changelog 0.3.0

### PR DESCRIPTION
**Pull request by psychocoderHPC**
The release is scheduled for 03.08.2021

### CODE REVIEW

**Review by j-stephan — APPROVED** [review]
Wow, we have release codenames?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,33 @@
...
+### Bug Fixes:
+- fix KernelWithElementLevel documentation #159
+- fix atomic function return type #163
+- fix ambiguous namespace `device` #167
+- fix ; warnings #172
+- fix multi GPU streams handling and default stream #183
+
+### New Features:
+- math functions: erf and pow #169
+- add support for Alpaka's OpenMP 5 and OpenACC backends #196
```
Comment (sbastrakov) [inline CHANGELOG.md:24]: `alpaka's` (not capitalized A)?

**Review by sbastrakov — APPROVED** [review]
One small thing above, but not essential

File: CHANGELOG.md (hunk 1 above)
Comment (psychocoderHPC) [reply CHANGELOG.md:24]: wr can keep it, I copied the description of the PR.

**Pull request merged**